rootfs/api/models/app.py

```diff
@@ -123,11 +123,18 @@ def create(self, *args, **kwargs):
         config = Config.objects.create(owner=self.owner, app=self)
         Release.objects.create(version=1, owner=self.owner, app=self, config=config, build=None)
 
+        # create required minimum resources in k8s for the application
+        self._scheduler.create(self.id)
+
+        # Attach the platform specific application sub domain to the k8s service
+        Domain(owner=self.owner, app=self, domain=self.id).save()
```

Comment:
Domain can now be created this early because `_scheduler.create` makes the base service k8s object